src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -791,11 +776,17 @@ const formConfig = {
           schema: applicantListSchema([], {
             titleSchema,
             'ui:description': blankSchema,
-            applicantRelationshipOrigin: customRelationshipSchema([
-              'blood',
-              'adoption',
-              'step',
-            ]),
+            applicantRelationshipOrigin: {
+              type: 'object',
+              properties: {
+                relationshipToVeteran: radioSchema([
+                  'blood',
+                  'adoption',
+                  'step',
+                ]),
+                otherRelationshipToVeteran: { type: 'string' },
+              },
+            },
```

Comment:
This schema was previously the only thing in `CustomRelationshipPattern.js`. It was only imported/used in this one location, so deleted that file and moved schema here.